src/components/Charts/parseResponse.js

```diff
@@ -132,12 +130,30 @@ const parseResponse = (state, province, response) => {
         total: sumBy(filterResponse, p => p.dwellings)
     };
 
-    const typesName = uniq(flatten(
-        filterResponse.map(p => p.dwellingTypes.map(type => type.type || DWELLINGS_WITHOUT_TYPE))
-    ));
+    const dwellingTypesNames = [];
+    const dwellingTypesValues = [];
+    const responseDwellingTypes = filterResponse.map(p => p.dwellingTypes);
+
+    const dwellingTypes = reduce(
+        responseDwellingTypes,
+        (result, item) => {
+            const newResult = {...result};
+            forEach(item, (value, key) => {
+                if (newResult[key]) {
+                    newResult[key] += value;
+                } else {
+                    newResult[key] = value;
+                }
+            });
+            return newResult;
+        },
+        {}
+    );
+
+    forEach(dwellingTypes, (value, key) => dwellingTypesNames.push(key) && dwellingTypesValues.push(value));
```

Comment:
Podés usar lodash: `keys` y `values`. Igualmente te conviene hacerlo en el `reduce` de arriba para ahorrarte una iteración.

Reply:
Se arreglo en https://github.com/indec-it/react-address-commons/pull/33